### DISCUSSION TITLE
Barry Arm Landslide, AK

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/Earth-Analytics-2023-01-Intro#89] [Awesome science]
Cool science...Amazing location...And I love working here, although I don't like to think about what would happen to us if the slide failed catastrophically while we are working there.  

[https://dggs.alaska.gov/hazards/barry-arm-landslide.html](url)

**Comment by ellenalamont17** [comment]
Alaska is such a beautiful and dynamic place to work. Not quite the same area, but I helped with some work outside of Glacier Bay NP, about two years back, looking for tsunami deposits using push cores. The Barry Arm slide story makes me think of a story I remember from a class discussion about whether tsunamis happen in the Atlantic Ocean. The professor mentioned the potential for a mega-tsunamic from a landslide failure in the Canary Islands based on geologic observations (e.g. https://www.e-education.psu.edu/earth107/node/1609) and the debate over whether it was possible (e.g. https://www.usgs.gov/observatories/hvo/news/volcano-watch-canary-islands-mega-tsunami-hypothesis-and-why-it-doesnt-carry)

**Reply by misterskye** [reply]
Thanks for sharing!!  Looking forward to looking into what you sent, and funny how I never really thought about tsunamis in the Atlantic.
